.circleci/config.yml

```
@@ -14,6 +14,12 @@ jobs:
       - image: quay.io/astronomer/ap-dind-golang:24.0.6
     steps:
       - test
+  integration-test:
+    machine:
+      resource_class: medium
+      image: ubuntu-2204:2024.11.1
```

```suggestion
      image: ubuntu-2204:current
```

nit: minor thing to not worry about regular circle ci releases.